felix/bpf-gpl/fib.h

```diff
@@ -212,7 +212,7 @@ static CALI_BPF_INLINE int forward_or_drop(struct cali_tc_ctx *ctx)
 		 * XXX those devices where we expect them before we even decap.
 		 */
 		if (CALI_F_FROM_HEP && state->tun_ip != 0 && ctx->fwd.mark != CALI_SKB_MARK_BYPASS_FWD) {
-			ctx->fwd.mark |= CALI_SKB_MARK_SKIP_RPF;
+			ctx->fwd.mark = CALI_SKB_MARK_SKIP_RPF;
```

Comment:
Why this change, might be worth a comment to say which flags you're deliberately clobbering here

Reply:
It was wrong to start with. It wasn't supposed to be ORed ever. NOw with RPF in BPF and not rendering any RPF chains in iptables, I think we should remove this mark combination completely. Will do in a separate PR.

Reply:
https://github.com/projectcalico/calico/pull/6085